src/defaults/definitions.js

```diff
@@ -3,40 +3,40 @@
 const lambdaNamespace = 'AWS/Lambda';
 
 module.exports = {
-  functionInvocations: {
-    namespace: lambdaNamespace,
-    metric: 'Invocations',
-    threshold: 100,
-    statistic: 'Sum',
-    period: 60,
-    evaluationPeriods: 1,
-    comparisonOperator: 'GreaterThanThreshold',
-  },
-  functionErrors: {
-    namespace: lambdaNamespace,
-    metric: 'Errors',
-    threshold: 10,
-    statistic: 'Maximum',
-    period: 60,
-    evaluationPeriods: 1,
-    comparisonOperator: 'GreaterThanThreshold',
-  },
-  functionDuration: {
-    namespace: lambdaNamespace,
-    metric: 'Duration',
-    threshold: 500,
-    statistic: 'Maximum',
-    period: 60,
-    evaluationPeriods: 1,
-    comparisonOperator: 'GreaterThanThreshold',
-  },
-  functionThrottles: {
-    namespace: lambdaNamespace,
-    metric: 'Throttles',
-    threshold: 50,
-    statistic: 'Sum',
-    period: 60,
-    evaluationPeriods: 1,
-    comparisonOperator: 'GreaterThanThreshold',
-  }
-};
\ No newline at end of file
+	functionInvocations: {
```

Comment:
Looks like whitespace changes. Can we please keep the existing linting rules in place.